src/introduction.tex

```diff
@@ -46,15 +46,17 @@
 
 \subsubsection{Statement of the problem}
 
-Nowadays the industry increasingly relies on cloud technologies. More and more companies start or move their products to cloud environment.
-It has been observed that gradually institutions introduce dedicated teams for managing the costs of cloud operations or use third-parties for that purpose. Such solutions cause great savings for everyone.
-Unfortunately existing toolkit is either very primitive or targets billion dollar companies. Currently neither Amazon nor any third-party offer a tool that would allow flexible, cheap and quick insight into the operation costs for small or middle-sized organizations.
+The software industry increasing adopt public cloud as the primary method of running their infrastructure.
```

Comment:
should this be `increasingly adopts`? I'm not sure but in this form the grammar doesn't make sense to me

Reply:
this is closer now, but still doesn't compile gramatically for me - we have `software industry` which has a role of an actor, a noun - and just after that the `increasing adoption of cloud resources` that also has a role of a noun - there's no correlation between those, we probably need a verb such as `is`, ie `software industry is increasing adoption`

Reply:
so - this might either just be missing that `is` there, or you mean something else